### DISCUSSION TITLE
code dir

### DISCUSSION THREAD

**Discussion by Okhayeeli** [scaffold-eth/scaffold-eth-2#845] [Q&A]
what is the code directory, vercel is asking for

**Comment by Pabl0cks** [comment]
Hello! 

I guess you're using `yarn vercel` to deploy your code to VErcel, if not let me know! 

Since Vercel probably detected the code in the first prompt at `\packages\nextjs` folder, when it prompts `In which directory is your code located?` you can leave default answer => `./`

If you still have problems deploying to Vercel, let us know! Closing this for now, feel free to reopen, or to send a message in [Scaffold-eth developers chat](https://t.me/joinchat/F7nCRK3kI93PoCOk)

**Reply by Okhayeeli** [reply]
thank you, i figured it out
